monai/losses/dice.py

```
@@ -239,11 +239,51 @@ class MaskedDiceLoss(DiceLoss):
+        smooth_dr: float = 1e-5,
+        batch: bool = False,
+        weight: Sequence[float] | float | int | torch.Tensor | None = None,
+        soft_label: bool = False,
+    ) -> None:
         """
         Args follow :py:class:`monai.losses.DiceLoss`.
         """
-        super().__init__(*args, **kwargs)
+
+        if sigmoid and softmax:
+            raise ValueError("Incompatible values: sigmoid=True and softmax=True.")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)